site/content/docs/development/architecture.md

```diff
@@ -0,0 +1,144 @@
+# Architecture and Concepts
+
+## Overview
+
+Cartographer is an open-sourced Supply Chain Choreographer for Kubernetes. Cartographer provides a set of Kubernetes
+controllers and CRDs that allow a platform operator to create an application platform by specifying Supply Chains and 
+application Delivery workflows.
+
+## Concepts
+
+### Blueprints
+
+Blueprints are a list of templates (called resources) and how they depend upon each other. It forms the dependency graph of
```

Comment:
A blueprint is a list of templates (called resources) that defines how the templates depend upon each other.

Reply:
please try and make suggested edits, then I can just click them